torchci/clickhouse_queries/queued_jobs/query.sql

```diff
@@ -1,6 +1,10 @@
 --- This query is used by HUD metrics page to get the list of queued jobs
 with possible_queued_jobs as (
-  select id, run_id from default.workflow_job where status = 'queued'
+  select id, run_id
+  from default.workflow_job -- FINAL not needed since we just use this to filter a table that has already been FINALed
+  where status = 'queued'
+    AND created_at < (CURRENT_TIMESTAMP() - INTERVAL 5 MINUTE)
+    AND created_at > (CURRENT_TIMESTAMP() - INTERVAL 1 WEEK)
```

Comment:
The max avg_queue_s in the queue_times_historical table is 1147999 which is 13 days.  Can we make this window bigger?  Even after extending it to 2 months the query takes still takes <1s to complete

Reply:
We could, but I'm wondering when it would actually matter.  Jobs are auto-cancelled after being queued for 3 days, aren't they?  

Those 13 day queued jobs are on `linux.20_04.4x` machines, which are github hosted runners.  The next highest is for linux.2xlarge with 8 days of queueing, which is surprising to see since it implies that we either don't cancel jobs after 3 days of queueing or github's job status api fails internally much more often than we realized.

Reply:
I assume its the jobs queued forever according to github.  If we don't need to care about these jobs that should have gotten cancelled but didn't actually, then the threshold can be lower